js/feedback.js

```diff
@@ -79,9 +79,9 @@ Feedback.prototype.submit = function(e) {
 
 Feedback.prototype.handleSuccess = function(response) {
   var message =
-    '<h2 class="feedback__title">Thank you for helping us improve betaFEC</h2>' +
+    '<h2 class="feedback__title">Thanks for helping us improve betaFEC</h2>' +
     '<p>This information has been reported on GitHub, where it is publicly visible. ' +
-    'You can track the status of your feedback <a target="_blank" href="' + response.html_url + '">here</a>.</p>';
+    '<a target="_blank" href="' + response.html_url + '">Track</a> the status of your feedback.</p>';
```

Comment:
@noahmanger, why isn't it working to make "Track" the link? Did I mess up the HTML?